src/main/resources/META-INF/plugin.xml

```diff
@@ -146,6 +146,17 @@
             implementationClass="com.jetbrains.snakecharm.inspections.SmkSubworkflowRedeclarationInspection"
     />
 
+    <localInspection
+            language="Snakemake" shortName="SmkUndefinedNameInspection"
+            enabledByDefault="true"
+            level="ERROR"
+            suppressId="SmkUndefinedNameInspection"
+            bundle="SnakemakeBundle"
+            groupKey="INSP.GROUP.snakemake"
+            key="INSP.NAME.undefined.name"
+            implementationClass="com.jetbrains.snakecharm.inspections.SmkUndefinedNameInspection"
+    />
+
     <psi.referenceContributor implementation="com.jetbrains.snakecharm.codeInsight.SnakemakeParamsInShellReferenceContributor"/>
 
```

Comment:
Html description file for the inspection is missing 
![image](https://user-images.githubusercontent.com/72933/60827700-74ac9580-a1b9-11e9-9db5-fd3a78c20ae4.png)